packages/app/pages/index.tsx

```diff
@@ -1,34 +1,57 @@
-import { ReactElement } from 'react'
-import { Button } from 'antd'
+import { ReactElement, useEffect } from 'react'
+import { Button, Spin } from 'antd'
+import useSWR from 'swr'
+import { API } from '@koh/api-client'
+import Router from 'next/router'
 
 export default function Home(): ReactElement {
-  return (
-    <div className="ml-auto mr-auto max-w-[500px] pt-10 text-center">
-      <div className="flex flex-col items-center justify-center">
-        <img src="/ubc_logo.png" alt="UBC logo" className="mb-4 mr-4 w-12" />
-        <h1>Welcome to HelpMe</h1>
+  // check if logged in
+  const { data, error } = useSWR(
+    `api/v1/profile`,
+    async () => await API.profile.index(),
+  )
+
+  // if logged in, redirect to courses page
+  useEffect(() => {
+    if (data && !error) {
+      // if data is not undefined and there's no error, it means the user is logged in
+      Router.push('/courses')
+    }
+  }, [data, error])
+
+  // only show landing page if not logged in
+  if (error?.response?.status === 401) {
```

Comment:
I think we should always show the page unless user is logged in?

Reply:
yep that's what this does

Reply:
How about status code 403?

Reply:
oh i suppose i can add that one. I was just confused on how 403 was handled in useProfile.ts as it logs the user out when the status code is 403, so I assumed maybe 403 was reserved for logging someone out or something. I will add 403

Reply:
or i'll try doing any error actually and see if that works, as it's probably a good idea to still show the page even if the login endpoint was down or something